### ISSUE TITLE
pkgbuilder -F does nothing

### ISSUE THREAD

**Issue by monotykamary** [Kwpolska/pkgbuilder#52]
`cower -d` works as expected. `pb -F` works as expected. `pkgbuilder -F` seemingly does nothing.

**Comment by Kwpolska** [comment]
Works for me:

```
$ pkgbuilder --version
PKGBUILDer v4.2.10
$ pkgbuilder -F python-nikola
:: Fetching package information...
:: Retrieving packages from aur...
(1/1) cloning python-nikola
Cloning into 'python-nikola'...
remote: Counting objects: 4, done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 4 (delta 0), reused 3 (delta 0)
Unpacking objects: 100% (4/4), done.
Successfully fetched: python-nikola
```

**Comment by monotykamary** [comment]
Might be an issue with my terminal then. I will do a hard reset on my settings.

**Comment by Kwpolska** [comment]
Which package are you trying to download?

**Comment by monotykamary** [comment]
Just testing and populating PKGBUILDS with `pkgbuilder -F $(pacman -Qqm)`.

**Comment by Kwpolska** [comment]
It might be a bit slow if you have many packages. There is a minor bug with unknown packages reporting, will get fixed in v4.2.11 (in a few minutes).
